assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -412,13 +467,15 @@ export function getAnalytics4MockResponse( args ) {
 									}
 								),
 								metricValues: [
-									...( rows[ 0 ]?.metricValues || [] ),
+									...( rows[ 1 ]?.metricValues || [] ),
 								],
 							},
 					  ]
 					: []
 			);
 
+			const firstItemIndex = rows.length - ( hasDateRange ? 2 : 1 );
```

Comment:
With a variable name like "first item" it'd be helpful to specify why we're not using a zero-based index here. It's just not evident from the code right around it.

Reply:
I've added a comment to explain the logic here. I've stuck with the name `firstItemIndex`, though, as, with the comment, I think it's still OK, as it refers to the first item within the context of `maximums` and `totals`, which I think should be evident from reading the code with the added context of the comment. However, if you still feel it's not clear, I'd be happy to change it. Open to suggestion on names :)